tests/Selector/Xpath/ManipulatorTest.php

```diff
@@ -56,8 +56,13 @@ public function getPrependedXpath()
             ),
             'multiline' => array(
                 'some_xpath',
-                "some_tag1 | some_tag2[@foo =\n 'bar|'']\n | some_tag3[foo | bar]",
-                "some_xpath/some_tag1 | some_xpath/some_tag2[@foo =\n 'bar|''] | some_xpath/some_tag3[foo | bar]",
+                "some_tag1 | some_tag2[@foo =\n 'bar|']\n | some_tag3[foo | bar]",
+                "some_xpath/some_tag1 | some_xpath/some_tag2[@foo =\n 'bar|'] | some_xpath/some_tag3[foo | bar]",
+            ),
+            'containing pipe' => array(
+                'some_xpath',
+                "some_tag[(contains(normalize-space(string(.)), 'foo|bar') | other_tag[contains(./@some_attribute, 'foo|bar')])]",
+                "some_xpath/some_tag[(contains(normalize-space(string(.)), 'foo|bar') | other_tag[contains(./@some_attribute, 'foo|bar')])]",
```

Comment:
well, that's the question: the original regex suggests that within square brackets expressions should not be prefixed. Are you saying that we should also change the 'complex condition' test case further above to produce a different result?